changelog.txt

```diff
@@ -14,6 +14,135 @@ Old-style release channel:
 * beta: version 2.3.12
 * stable: version 2.3.12
 
+Unstable development rolling releases:
+* devel: version 2.4.0
+
+2.4.0: devel
+
+Highlights:
+* Major 3rd party software updates (e.g. updated Alpine, Arkime, Zeek, Suricata, lot of libraries)
+* User Management support
+* PassiveDNS search UI
+* PortDiff improvements (e.g. baseline, banners)
+* Major performance improvements (TrafficGuard, PassiveDNS)
+
+- Bug fixes:
+  [SF-2435] - Suricata instrument: Suricata classification config is overwritten
+  [SF-2428] - Suricata instrument: Suricata tries to create trigger message from event with no context_uuid and fails
+  [SF-2427] - API: Time-critical transient event deliveries can be delayed in certain scenarios
+  [SF-2421] - Sensor Orchestrator, UI: get_physical_interfaces does not update "user" field immediately
+  [SF-2418] - UI: Fix instrument download editing
+  [SF-2417] - API: Fix download validation in SensorConfig schema
+  [SF-2412] - PortDiff Instrument: PortDiff: Exposed services descriptions have trailing spaces
+  [SF-2405] - UI: Sensor/FM UI caches a failed configuration change
+  [SF-2400] - SensorFu Beacon: SensorFu Beacon output events do not work
+  [SF-2398] - TrafficGuard Manager: Fix autovacuum and autoanalyze for tgmanager
+  [SF-2395] - FleetCert: python3.8 missing from sensor in some installations
+  [SF-2394] - PortDiff Instrument: Banner diff event does not contain all removed banenrs
+  [SF-2375] - Sensor Orchestrator: Sensor orchestrator network thread crash while upgrading
+  [SF-2360] - UI: Instrument custom config UI boolean values are wrong
+  [SF-2353] - API: Subscription worker _send_worker() is not cancelled properly on all cases
```

Comment:
Perhaps no need to add internal method names to a public changelog, they don't add any value.